Src/IronPython/Runtime/PythonBuffer.cs

```diff
@@ -238,20 +223,16 @@ public PythonBuffer(CodeContext/*!*/ context, object @object, int offset)
         }
 
         public int __len__() {
-            return Math.Max(_size, 0);
+            return Math.Max(Size, 0);
         }
 
-        internal int Size {
-            get {
-                return _size;
-            }
-        }
+        internal int Size { get; private set; }
```

Comment:
Maybe not worth the effort, but it seems like `_object` and `Size` should be readonly.